<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Copyright 2010, Martin Koller</author>
 <class>QRKGastroFinishTicketDialog</class>
 <widget class="QWidget" name="QRKGastroFinishTicketDialog">
  <property name="modal" stdset="0">
   <bool>false</bool>
  </property>
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Bezahlen</string>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="accessibleName">
   <string notr="true"/>
  </property>
  <property name="accessibleDescription">
   <string notr="true"/>
  </property>
  <property name="windowFilePath">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="icon">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>:/icons/arrow-left.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="3">
        <widget class="QrkPushButton" name="debitcardReceipt">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>Bankomat</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:src/icons/bankomat.png</normaloff>:src/icons/bankomat.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QrkPushButton" name="cashReceipt">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>BAR</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:src/icons/money48.png</normaloff>:src/icons/money48.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QrkPushButton" name="advertisingTicket">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>Werbung</string>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QrkPushButton" name="employeeTicket">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>Mitarbeiter</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QrkPushButton" name="privateTicket">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>Privat</string>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QrkPushButton" name="creditcardReceipt">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>Kreditkarte</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:src/icons/paymentmodes48.png</normaloff>:src/icons/paymentmodes48.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QrkPushButton" name="cancel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>Abbrechen</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/icons/cancel.png</normaloff>:/icons/cancel.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QrkPushButton</class>
   <extends>QPushButton</extends>
   <header>qrkpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cashReceipt</tabstop>
  <tabstop>debitcardReceipt</tabstop>
  <tabstop>creditcardReceipt</tabstop>
  <tabstop>privateTicket</tabstop>
  <tabstop>employeeTicket</tabstop>
  <tabstop>advertisingTicket</tabstop>
  <tabstop>cancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>QRKGastroFinishTicketDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>477</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
